lib/content-services/src/lib/search/components/search-date-range/search-date-range.component.spec.ts

```diff
@@ -62,39 +65,39 @@ describe('SearchDateRangeComponent', () => {
         expect(component.form).toBeDefined();
     });
 
-    it('should setup the format of the date from configuration', () => {
-        component.settings = { field: 'cm:created', dateFormat: dateFormatFixture };
+    it('should check the format of the date from component', () => {
+        component.settings = { field: 'cm:created', dateFormat: DateFnsUtils.convertMomentToDateFnsFormat(dateFormatFixture) };
         fixture.detectChanges();
 
-        expect(adapter.overrideDisplayFormat).toBe(dateFormatFixture);
+        expect(component.datePickerFormat).toBe(DateFnsUtils.convertMomentToDateFnsFormat(dateFormatFixture));
```

Comment:
this test makes no sense at all... you set the "component.settings.dateFormat" above, and then test it is the same value

Reply:
Changed the test